src/ui/server/helpers/mappings/map-currencies.ts

```diff
@@ -0,0 +1,32 @@
+import mapSelectOption from './map-select-option';
+import sortArrayAlphabetically from '../sort-array-alphabetically';
+import { Currency } from '../../../types';
+
+/**
+ * mapCurrencies
+ * Map all currencies into the required structure for GOV select component.
+ * @param {Array} Array of currency objects
+ * @returns {String} Selected currency
+ * @returns {Array} Array of mapped currencies, ordered alphabetically
+ */
+const mapCountries = (currencies: Array<Currency>, selectedValue?: string) => {
+  const mapped = currencies.map(({ name, isoCode }) => mapSelectOption(name, isoCode, true, selectedValue));
+
+  const sortedCurrencies = sortArrayAlphabetically(mapped, 'text');
```

Comment:
I'm not sure if this is redundant - 

The API is returning them alphabetically so I don't think we need this line?

The file is called `map-currencies` but the function is called `mapCountries`?

We have these functions:

```js
map-currencies/as-radio-options
map-currencies/as-select-options
```

If this function is _not_ redundant, seems we should update `mapCurrenciesAsSelectOptions` instead of creating a third mapping function